packages/css-library/src/stylesheets/formation-overrides/_variables.scss

```diff
@@ -0,0 +1,113 @@
+/**
+  Variables ported over from Formation so that we can keep those stylesheets
+  working while we work on deprecation.
+**/
+
+@import '../override-function';
+
+$font-path: "./fonts";  // TODO: Verify asset path
+$image-path: "./img"; // TODO: Verify asset path
+
+$label-border-radius: 2px;
+$h5-font-size: scale-rem(1.5rem);
+
+$color-black-light: #212121;
+$color-base: $color-black-light;
```

Comment:
These are all "imposter tokens". They are only here so that the ported stylesheets continue to work. These should all go away as we remove things.